src/commonMain/kotlin/fr/acinq/phoenixd/Api.kt

```diff
@@ -204,6 +204,14 @@ class Api(
                     }
                     call.respond(GeneratedInvoice(invoice.amount?.truncateToSatoshi(), invoice.paymentHash, serialized = invoice.write()))
                 }
+                post("createoffer") {
+                    val formParameters = call.receiveParameters()
+                    val amount = formParameters.getOptionalLong("amountSat")?.sat
+                    val maxDescriptionSize = 128
+                    val description = formParameters["description"]
+                        ?.also { if (it.length > maxDescriptionSize) badRequest("Request parameter description is too long (max $maxDescriptionSize characters)") }
```

Comment:
You should also check that if there is an amount, there is also a description.

Reply:
Thanks! Done in743c212.